appveyor.yml

```diff
@@ -5,6 +5,13 @@ image:
   - Ubuntu
 configuration: Release
 test: off
+install:
+- ps: $RequiredDotnetVersion =  $(Get-Content ./global.json | ConvertFrom-Json).sdk.version
+- ps: $env:DOTNET_INSTALL_DIR = "$pwd\.dotnetsdk"
+- ps: mkdir $env:DOTNET_INSTALL_DIR -f | Out-Null
```

Comment:
@nblumhardt It does not work.

Reply:
On the Ubuntu target it might be nicer to use `apt` 🤔

Reply:
@nblumhardt ps via `dotnet-install.ps1` and the `global.json` is way i have seen it done everywhere

`RequiredDotnetVersion ` should not be required. u can point `dotnet-install.ps1` to a `global.json`:

```
./dotnet-install.ps1 -JSonFile src/global.json
```

FWIW here is the `build_script` portion i use in many projects that use appveyor 

```
build_script:
- pwsh: |
    if ($isWindows) {
      Invoke-WebRequest "https://dot.net/v1/dotnet-install.ps1" -OutFile "./dotnet-install.ps1"
      ./dotnet-install.ps1 -JSonFile src/global.json -Architecture x64 -InstallDir 'C:\Program Files\dotnet'
    }
    if ($isMacOS) {
      Invoke-WebRequest "https://dot.net/v1/dotnet-install.sh" -OutFile "./dotnet-install.sh"
      sudo chmod u+x dotnet-install.sh
      sudo ./dotnet-install.sh --jsonfile src/global.json --architecture x64 --install-dir '/usr/local/share/dotnet'
    }
    if ($isLinux) {
      Invoke-WebRequest "https://dot.net/v1/dotnet-install.sh" -OutFile "./dotnet-install.sh"
      sudo chmod u+x dotnet-install.sh
      sudo ./dotnet-install.sh --jsonfile src/global.json --architecture x64 --install-dir '/usr/share/dotnet'
    }
```